src/components/AdminDashboard/SightedListedSpecies.jsx

```diff
@@ -1,26 +1,28 @@
 import { React } from 'react';
-import { Text, Grid, GridItem } from '@chakra-ui/react';
+import { Text, Grid, GridItem, Box, Flex } from '@chakra-ui/react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Grid' is defined but never used.